assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -75,89 +77,60 @@ function mapDispatchToProps(dispatch) {
 
 const DirectDebitForm = (props: PropTypes) => (
   <div className="component-direct-debit-form">
-    <SortCodeInput
-      onChange={props.updateSortCode}
-      value={props.sortCode}
+
+    <AccountHolderNameInput
+      onChange={props.updateAccountHolderName}
+      value={props.accountHolderName}
     />
 
     <AccountNumberInput
       onChange={props.updateAccountNumber}
       value={props.accountNumber}
     />
 
-    <AccountHolderNameInput
-      onChange={props.updateAccountHolderName}
-      value={props.accountHolderName}
+    <SortCodeInput
+      onChange={props.updateSortCode}
+      sortCodeArray={props.sortCodeArray}
     />
 
     <ConfirmationInput
       onChange={props.updateAccountHolderConfirmation}
       checked={props.accountHolderConfirmation}
     />
 
-    <ErrorMessage message={props.formError} />
     <button
       id="qa-pay-with-direct-debit-pay"
       className="component-direct-debit-form__pay-button"
       onClick={() => props.payDirectDebitClicked(props.callback)}
     >
-      Pay with Direct Debit
+      <SvgDirectDebitSymbol />
+      <span>Contribute with Direct Debit</span>
+      <SvgArrowRightStraight />
     </button>
 
-    <div className="component-direct-debit-form__legal__title">
-      Advance notice
-    </div>
+    <ErrorMessage message={props.formError} svg={<SvgExclamationAlternate />} />
 
     <div className="component-direct-debit-form__legal__content">
-      <p>The details of your Direct Debit instruction including payment schedule, due date,
-        frequency and amount will be sent to you within three working days. All the normal
-        Direct Debit safeguards and guarantees apply.
-      </p>
-      <p>
-        Your payments are protected by the&nbsp;
-        <a target="_blank" rel="noopener noreferrer" href="https://www.directdebit.co.uk/DirectDebitExplained/pages/directdebitguarantee.aspx">
+      <p><strong>Advance notice</strong> The details of your Direct Debit instruction including
```

Comment:
I wonder if this one could be another component. For example `legalNotice`

Reply:
Yup might be nicer